command/cli.go

```diff
@@ -66,6 +66,7 @@ type MetaArgs struct {
 	// TODO(azr): in the future, I want to allow passing multiple path to
 	// merge HCL confs together; but this will probably need an RFC first.
```

Comment:
Since this PR resolves Adrien's (azr)'s old TODO, you should remove this TODO :)!